src/or/or.h

```diff
@@ -959,6 +959,8 @@ typedef enum {
 #define CELL_CREATE2 10
 #define CELL_CREATED2 11
 #define CELL_PADDING_NEGOTIATE 12
+#define CELL_CREATE2V 13
```

Comment:
These values should be in the 128..255 range, since they are new variable-length cells.